csrankings-h.csv

```diff
@@ -525,6 +525,8 @@ Helen Wright Hastie,University of Edinburgh,http://www.inf.ed.ac.uk/people/staff
 Helen Yannakoudakis,King's College London,https://www.kcl.ac.uk/people/helen-yannakoudakis,B3WGvRsAAAAJ
 Helena Cristina da Gama Leitão,UFF,http://www2.ic.uff.br/~hcgl,NOSCHOLARPAGE
 Helena Galhardas,Universidade de Lisboa,http://web.ist.utl.pt/helena.galhardas,oBY_WrMAAAAJ
+Helena Holmström,Malmö University,https://mau.se/en/persons/helena.holmstrom.olsson/,bjGw_5QAAAAJ

```

Comment:
Not listed in CS dept.

Reply:
Similar case as Martin Host.
Please refer to https://github.com/emeryberger/CSrankings/pull/6822#discussion_r1667619091